packages/strapi-plugin-content-manager/admin/src/components/SelectWrapper/index.js

```diff
@@ -79,14 +79,22 @@ function SelectWrapper({
         return;
       }
 
-      const params = { _limit: state._limit, _start: state._start, ...defaultParams };
+      setIsLoading(true);
+
+      const params = { _limit: state._limit, ...defaultParams };
 
       if (state._contains) {
         params[containsKey] = state._contains;
       }
 
       try {
-        const data = await request(endPoint, { method: 'GET', params, signal });
+        const idsToOmit = value ? value.map(val => val.id) : [];
```

Comment:
This won't work in case of oneToOne relations, can you make the following update:

```
 const idsToOmit = useMemo(() => {
    if (!value) {
      return [];
    }

    if (isSingle) {
      return [value];
    }

    return value.map(val => val.id);
  }, [isSingle, value]);
```